site/frontend/templates/pages/dashboard.html

```diff
@@ -1,21 +1,36 @@
 {% extends "layout.html" %}
+{% block head %}
+<style>
+    .graphs {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+
+        @media screen and (max-width: 768px) {
+            grid-template-columns: 1fr;
+        }
+    }
+</style>
+{% endblock %}
 {% block content %}
 <details style="margin-top: 10px;">
-<summary>What data is in the dashboard?</summary>
+    <summary>What data is in the dashboard?</summary>
 
-The dashboard shows performance results for all stable Rust releases going back to
-<code>1.28.0</code>, along with the latest <code>beta</code> release. The displayed
-duration is an arithmetic mean amongst all
-<a href="https://github.com/rust-lang/rustc-perf/tree/master/collector/compile-benchmarks#stable">stable</a>
-benchmarks.
+    The dashboard shows performance results for all stable Rust releases going back to
+    <code>1.28.0</code>, along with the latest <code>beta</code> release. The displayed
+    duration is an arithmetic mean amongst all
+    <a href="https://github.com/rust-lang/rustc-perf/tree/master/collector/compile-benchmarks#stable">stable</a>
+    benchmarks.
```

Comment:
We could mention the runtime benchmarks too. Something like "The dashboard also shows the average duration of runtime benchmarks, which measure the performance of Rust programs compiled by a given version of the Rust compiler.".